lib/jsonapi_parameters/version.rb

```diff
@@ -1,5 +1,5 @@
 module JsonApi
   module Parameters
-    VERSION = '2.0.0'.freeze
+    VERSION = '2.1.0'.freeze
```

Comment:
I'm not sure if you should change patch instead of minor one 🤔

Reply:
Yeah, I was wondering too. From the summary at [semver.org](https://semver.org/): 

MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.

So this is not a bug fix, its more of an enhancement that is backwards compatible. While the change does not influence most of the users, it cleared out a path on how the default handlers should be defined, and made the code more readable, while also using things built into Ruby.

Because of that I decided to go with MINOR version instead of PATCH. Maybe PATCH is better?

Reply:
I understand your point of view and I've looked at it more from a business value side to the end user of the library.

I agree though with your justification, and resolving this one 🙂